tests/unit/core/TestInstallAddon.js

```diff
@@ -1322,7 +1385,7 @@ describe(`${__filename}: withInstallHelpers`, () => {
       return uninstall(addon).then(() => {
         sinon.assert.calledWith(fakeTracking.sendEvent, {
           action: TRACKING_TYPE_THEME,
-          category: UNINSTALL_CATEGORY,
+          category: getAddonEventCategory(addon.type, UNINSTALL_ACTION),
```

Comment:
Elsewhere you have used an explicit type constant (e.g., `ADDON_TYPE_THEME`) as opposed to `addon.type` and I think it would be good to do that here as well, in case something is wrong with the test setup and you don't end up with the type of `addon` you expect.

Reply:
yep, good catch.. thanks : )